pkg/controllers/cloud/cloudentityselector_controller.go

```diff
@@ -133,6 +121,27 @@ func (r *CloudEntitySelectorReconciler) updatePendingSyncCountAndStatus() {
 	}
 }
 
+// waitForPollDone waits until account poller has polled the cloud inventory or timeout is reached.
+func (r *CloudEntitySelectorReconciler) waitForPollDone(accountNamespacedName *types.NamespacedName) error {
+	pollerScope, err := r.Poller.getAccountPoller(accountNamespacedName)
+	if err != nil {
+		return err
+	}
+
+	r.Log.Info("Poll cloud inventory in a synchronous call", "account", *accountNamespacedName)
+	if err = wait.PollImmediate(100*time.Millisecond, 2*time.Second, func() (done bool, err error) {
```

Comment:
take timeout for polling to be 1 min, as input too

Reply:
Changed the poll timeout to 1 minute. Will update the function to specify an argument

Reply:
Done